test/controllers/ControllerSpecBase.scala

```diff
@@ -29,5 +32,15 @@ trait ControllerSpecBase extends SpecBase {
 
   def dontGetAnyData: FakeDataRetrievalAction = new FakeDataRetrievalAction(None)
 
+  def getMandatoryEstablisher: FakeDataRetrievalAction = new FakeDataRetrievalAction(Some(
+    Json.obj(
+      CompanyDetailsId.toString ->
+        CompanyDetails("Test Company Name", None, None),
+      "directors" -> Json.arr(
+        Json.obj(
+          DirectorDetailsId.toString -> DirectorDetails("test first name", Some("test middle name"), "test last name", LocalDate.now())
+        )
+      )
+    )))
 
```

Comment:
I don't think getMandatoryEstablisher is the correct name for this method. It's returning company details and directors not an establisher.